career/360-review.md

```diff
@@ -0,0 +1,38 @@
+# 360 Reviews
+
+ A 360 review is a tool for gathering feedback for an individual (evaluee). It is done by interviewing people who work with the evaluee in different contexts or with different roles.
+
+## Goal
+
+The ultimate goal is growth, and it is achieved by reflection.
+
+360 reviews provide in-depth feedback for the evaluee so that they can reflect on how others perceive their work. It is usually the most extensive feedback that the evaluee receives in the year.
+
+The 360 review is not a tool for determining raises or promotions. Its sole focus is on providing feedback.
+
+## Preparation
+
+The preparation consists of getting to know which areas the evaluee wants feedback in.
+Set up a short interview 1 or 2 weeks in advance. Be clear about the intention of the meeting in the agenda. Make the evaluee prepare a list of questions. During the interview, dig a bit deeper into the list. Why do they want to know that? Who do they want to hear from? The list may include a close client and their manager. Is there anything that's not on the list but may be interesting? Use your experience from other 360 reviews to suggest what the evaluee may want to know.
+
+
+## Conducting the interviews
+
+Once we have the list, set up interviews with the people who work close to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
```

Comment:
```suggestion
Once we have the list, set up interviews with the people who work close to the evaluee. Invite them 1 or 2 weeks in advance and let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
```